torch/csrc/api/include/torch/nn/modules/batchnorm.h

```diff
@@ -24,7 +24,10 @@ class BatchNormImpl : public torch::nn::Cloneable<BatchNormImpl> {
   explicit BatchNormImpl(BatchNormOptions options);
 
   void reset() override;
-  std::vector<Variable> forward(std::vector<Variable>);
+
+  Variable forward(Variable input);
+  Variable pure_forward(Variable input, Variable mean, Variable variance);
```

Comment:
It's because of `Sequential`. `AnyModule` can't handle default arguments, so we need one overload that takes a variable and produces a variable in order to pack it into `Sequential`